pkg/ccl/changefeedccl/changefeed_processors.go

```diff
@@ -1389,9 +1386,6 @@ func (cf *changeFrontier) forwardFrontier(resolved jobspb.ResolvedSpan) error {
 		return err
 	}
 
-	if log.V(1) {
-		log.Infof(cf.Ctx(), "forwarding frontier. changed: %t, resolved: %s", frontierChanged, resolved)
```

Comment:
Curious why the infos are no longer useful? They were probably instrumented for the sake of finding this race, but perhaps they might be useful for finding other regressions?

Reply:
The logging I added is a bit messy. I think I'll add neater ones later.

Reply:
TYFR